apps/website/src/components/modules/UserProfile/UserProfile.js

```diff
@@ -18,6 +18,7 @@ import { Misc, UserInfo, UserSection, Wrapper } from './StyledUserProfile';
 import UserInterests from './UserInterests';
 import { useRouter } from "next/router";
 // import DiscordSection from "./DiscordSection/DiscordSection";
+import { cleanDataList } from '@devlaunchers/ideaspace/src/utils/StrapiHelper';
```

Comment:
I remember this function, I think it has to go to the utility shared package, because apps shouldn't depend on each other

Reply:
Does this require changing the function to type script? I'm not sure how to proceed with that.

Reply:
@dbradham no, only moving it to the utility folder and exporting it